src/ClientData/DataManagerTest.cpp

```diff
@@ -115,6 +116,25 @@ TEST(DataManager, CanOnlyBeUsedFromTheMainThread) {
                                             WineSyscallHandlingMethod::kNoSpecialHandling);
   CallMethodOnDifferentThreadAndExpectDeath(data_manager,
                                             &DataManager::wine_syscall_handling_method);
+  CallMethodOnDifferentThreadAndExpectDeath(data_manager, &DataManager::GetTimeRangeSelectionStart);
+  CallMethodOnDifferentThreadAndExpectDeath(data_manager, &DataManager::GetTimeRangeSelectionEnd);
+  CallMethodOnDifferentThreadAndExpectDeath(data_manager, &DataManager::ClearTimeRangeSelection);
+  CallMethodOnDifferentThreadAndExpectDeath(data_manager, &DataManager::SetTimeRangeSelection, 0,
+                                            5);
+}
+
+TEST(DataManager, TimeRangeSelection) {
```

Comment:
This should be three separate test cases, as you test three behaviours -- default state, interval update and clearing the inetrval.

Reply:
done.